### DISCUSSION TITLE
Limiting the -z-axis

### DISCUSSION THREAD

**Discussion by ghost** [google/model-viewer#3161] [Q&A] [answered]
I'm using a model-viewer as a display for a 3D render of a car. I would like for the users to be able to rotate the car but be unable to rotate the camera to the lower part of the vehicle. Basically, I want the camera to move left-right-up while being able to move down(-z-axis) but not so far down as to where you can see the underneath of the car.

**Comment by samaneh-kazemi** [comment, marked as answer]
Hi @Archoniv, thanks for reaching out. Setting max camera orbit for phi to 90 degree is exactly horizontal and you can work with that. 
``` max-camera-orbit = "auto 90deg auto" ```
Checkout our documents for more details on [max-camera-orbit](https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-maxCameraOrbit). Hope this helps.

**Reply by ghost** [reply]
Thank you, that's what I was looking for.
